gateway/http.d/apicast.conf.liquid

```diff
@@ -14,6 +14,13 @@ map "" $extended_access_logs_enabled {
   default '0';
 }
 
+map "" $post_action_impact {
```

Comment:
Shouldn't be removed from https://github.com/3scale/APIcast/blob/e17e421944bdffe97366e306123a83c2874720dd/gateway/conf.d/apicast.conf#L159 ?

Reply:
currently investigating this. Removing this line would make sense but it seems to require reviewing our non-blackbox tests (that don't use apicast.conf.liquid).

Reply:
non blackbox tests conversion to Blackbox tracked here: https://github.com/3scale/APIcast/pull/1343